test/TestMain.hx

```diff
@@ -30,10 +30,10 @@ class TestMain
 			var httpClient = new HTTPClient(new SummaryReportClient());
 		#end
 
-		var runner:TestRunner = new TestRunner(client); 
+		var runner:TestRunner = new TestRunner(client);
 		runner.addResultClient(httpClient);
-		//runner.addResultClient(new HTTPClient(new JUnitReportClient()));
-		
+		runner.addResultClient(new HTTPClient(new JUnitReportClient()));
```

Comment:
I believe this still poses an issue? Although not with this smaller number of tests.

https://github.com/massiveinteractive/MassiveUnit/wiki/Known-Issues#http-post-causes-nekotools-server-to-hang